frontend/src/metabase/public/components/widgets/SharingPane.tsx

```diff
@@ -0,0 +1,208 @@
+                  onDisablePublicLink();
+                }}
+              >
+                <Toggle value={true} />
+              </Confirm>
+            ) : (
+              <Toggle
+                value={false}
+                onChange={() => {
+                  MetabaseAnalytics.trackStructEvent(
+                    "Sharing Modal",
+                    "Public Link Enabled",
+                    resourceType,
+                  );
+                  onCreatePublicLink();
+                }}
+              />
+            )}
+          </div>
+        </div>
+      )}
+      <div
+        className={cx("mb4 flex align-start", {
+          disabled: !resource.public_uuid,
+        })}
+      >
+        <div
+          style={{ width: 98, height: 63 }}
+          className="bordered rounded shadowed flex layout-centered"
+        >
+          <Icon name="link" size={32} />
+        </div>
+        <div className="ml2">
+          <OptionHeader className="text-brand">{t`Public link`}</OptionHeader>
+          <Description className="mb1">{t`Share this ${resourceType} with people who don't have a Metabase account using the URL below:`}</Description>
+          <CopyWidget value={publicLink} />
+          {extensions && extensions.length > 0 && (
+            <div className="mt1">
+              {extensions.map(extension => (
+                <span
+                  key={extension}
+                  className={cx(
+                    "cursor-pointer text-brand-hover text-bold text-uppercase",
+                    extension === extensionState ? "text-brand" : "text-light",
+                  )}
+                  onClick={() =>
+                    setExtension(extensionState =>
+                      extension === extensionState ? null : extension,
+                    )
+                  }
+                >
+                  {extension}{" "}
+                </span>
+              ))}
+            </div>
+          )}
+        </div>
+      </div>
+      <div
+        className={cx("mb4 flex align-start", {
+          disabled: !resource.public_uuid,
+        })}
+      >
+        <img
+          width={98}
+          src="app/assets/img/simple_embed.png"
+          srcSet="
+              app/assets/img/simple_embed.png     1x,
+              app/assets/img/simple_embed@2x.png  2x
+            "
+        />
+        <div className="ml2">
+          <OptionHeader className="text-green">{t`Public embed`}</OptionHeader>
+          <Description className="mb1">{t`Embed this ${resourceType} in blog posts or web pages by copying and pasting this snippet:`}</Description>
+          <CopyWidget value={iframeSource} />
+        </div>
+      </div>
+      <div
+        className={cx("mb4 flex align-start", {
+          disabled: shouldDisableEmbedding,
+          "cursor-pointer": !shouldDisableEmbedding,
+        })}
+        onClick={() => {
+          if (!shouldDisableEmbedding) {
+            console.log("wow man");
```

Comment:
😮 👨  ?

Reply:
dayum, that's a leftover from troubleshooting 🙇